### PR TITLE
Report operand version using environment variable set in operator deployment

### PR DESCRIPTION
**Pull request by mfojtik**
builds on https://github.com/openshift/library-go/pull/326

### CODE REVIEW

File: manifests/0000_10_kube-apiserver-operator_06_deployment.yaml [hunk 1]
```diff
@@ -43,6 +43,8 @@ spec:
           value: quay.io/openshift/origin-hypershift:v4.0
         - name: OPERATOR_IMAGE
           value: docker.io/openshift/origin-cluster-kube-apiserver-operator:v4.0
+        - name: OPERAND_IMAGE_VERSION
+          value: "0.0.1-snapshot_openshift"
```
Comment (deads2k) [inline manifests/0000_10_kube-apiserver-operator_06_deployment.yaml]: `0.0.1-snapshot-kubernetes` isn't it?

File: manifests/0000_10_kube-apiserver-operator_06_deployment.yaml [hunk 2]
```diff
@@ -43,6 +43,8 @@ spec:
           value: quay.io/openshift/origin-hypershift:v4.0
         - name: OPERATOR_IMAGE
           value: docker.io/openshift/origin-cluster-kube-apiserver-operator:v4.0
+        - name: OPERAND_IMAGE_VERSION
+          value: "0.0.1-snapshot_kubernetes"
```
Comment (deads2k) [inline manifests/0000_10_kube-apiserver-operator_06_deployment.yaml]: @sanchezl write us a test that makes sure this gets substituted properly and ends up as `1.XX.X`  do not hard code a number.

**Comment by wking** [comment]
[e2e-aws][1]:

```
level=fatal msg="failed to initialize the cluster: Cluster operator kube-apiserver is still updating: upgrading kube-apiserver from 0.0.1-2019-04-03-151049_kubernetes to 1.13.4: timed out waiting for the condition"
```

And in the most recent [e2e-aws-operator][2] too, so probably not just a flake?

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/366/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws/1867
[2]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/366/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-aws/1866/

File: manifests/0000_10_kube-apiserver-operator_06_deployment.yaml (hunk 2 above)
Comment (sanchezl) [reply manifests/0000_10_kube-apiserver-operator_06_deployment.yaml]: Opened https://github.com/openshift/cluster-kube-apiserver-operator/pull/367

**Pull request closed without merging**